crates/biome_js_analyze/src/lint/suspicious/no_explicit_any.rs

```diff
@@ -26,7 +26,7 @@ declare_rule! {
     ///
     /// ```ts,expect_diagnostic
     /// class SomeClass {
-    ///   message: Array<Array<any>>;
```

Comment:
This is probably a result of `find and replace` that breaks the tests in the website repo:

https://github.com/biomejs/website/actions/runs/9109965581/job/25043948850#step:8:378